### DISCUSSION TITLE
Datacamp renewal error

### DISCUSSION THREAD

**Discussion by Kavyaraikar21** [community/community#145988] [GitHub Education] [answered]
### Select Topic Area

Question

### Body

Hi GitHub Support Team,

I was verified as a student on July 2, 2024. My GitHub Student Developer Pack is still active, but I recently noticed that my 3-month Datacamp subscription, which was included in the pack, has expired.

I would like to know if it is possible to reapply for the GitHub Student Developer Pack or renew the Datacamp benefit to continue my learning journey. If reapplication is not possible at this time, could you kindly suggest any alternatives or the best course of action to regain access to Datacamp?
![Screenshot 2024-12-01 225517](https://github.com/user-attachments/assets/770d579a-c81c-4160-949f-e8fc9adbf1ec)

**Comment by hassamohammed** [comment, marked as answer]
You can only apply for the GitHub Student Developer Pack once every 12 months, so reapplying before a year has passed is not possible. Since your GitHub Student Developer Pack benefits, including the DataCamp subscription, expired, you'll need to wait until your eligibility to reapply.

As an alternative, here are a few suggestions:

Look for Promotions: DataCamp frequently offers discounts or promotions, especially for students. Keep an eye out for those or subscribe to their newsletter for updates.

Explore Other Free Learning Platforms: There are free courses and resources available that can continue your learning journey:

Kaggle: Offers many free data science courses and competitions.
Coursera: Offers free courses in data science, especially from universities.
edX: Provides free courses in partnership with top universities and companies.
Udacity: Occasionally offers free courses in data science and related fields.
University/Institutional Access: If you're part of a university or college, check if they provide DataCamp access or other similar platforms for free through their partnerships.

Free Trial Options: DataCamp often provides limited-time free trials to new users. You could sign up with a new email address to access these trials.

GitHub Student Pack Expiration: When your GitHub Student Pack becomes eligible for reapplication in the future, you could reapply and regain access to the benefits, including DataCamp.

Hope this helps.

**Reply by Kavyaraikar21** [reply]
Okay thank yu
